services/app-api/handlers/health_check.ts

```diff
@@ -1,14 +1,38 @@
 import { APIGatewayProxyHandler } from 'aws-lambda'
+import LaunchDarkly from 'launchdarkly-node-server-sdk'
+
+const ldClientKey = process.env.LD_SDK_KEY ?? ''
+if (ldClientKey === '') {
+    throw new Error('LD_SDK_KEY environment variable is not set')
+}
 
 export const main: APIGatewayProxyHandler = async () => {
+    const ldClient = LaunchDarkly.init(ldClientKey)
+    await ldClient.waitForInitialization()
```

Comment:
Do we know what happens here if LD doesn't respond?

Reply:
Ah yeah, it'll promise reject -- I just pushed a change to try/catch it.